tests/runtest.py

```diff
@@ -1,12 +1,36 @@
 #!/usr/bin/env python
-################################################################################
-################################################################################
 #
-# Module: runtest.py
+## Licensed to the .NET Foundation under one or more agreements.
+## The .NET Foundation licenses this file to you under the MIT license.
+## See the LICENSE file in the project root for more information.
+#
+##
+# Title               :runtest.py
 #
 # Notes:
 #  
-# Universal script to setup and run the xunit msbuild test runner.
+# Universal script to setup and run the xunit console runner. The script relies 
+# on runtest.proj and the bash and batch wrappers. All test excludes will also 
+# come from issues.targets. If there is a jit stress or gc stress exclude, 
+# please add GCStressIncompatible or JitOptimizationSensitive to the test's 
+# ilproj or csproj.
+#
+# The xunit runner currently relies on tests being built on the same host as the
+# target platform. This requires all tests run on linux x64 to be built by the
+# same platform and arch. If this is not done, the tests will run correctly;
```

Comment:
Won't this cause the official build test run to fail? If I recall, these would build wrappers on windows and run them on linux.

Reply:
This is not necessarily true for helix, which currently gets around this issue by setting an env variable BuiltAgainstPackages.